components/Cards/Card/Card.js

```diff
@@ -19,7 +18,14 @@ function Card({ children, className, hasAnimationOnHover, ...props }) {
 
   return (
     <article
-      className={classNames(styles.Card, className, { [styles.animatedCard]: hasAnimationOnHover })}
+      className={classNames(
+        'items-center bg-white text-secondary flex flex-col flex-nowrap justify-around m-4 min-h-[100px] min-w-[100px] p-6 shadow-md focus-visible:outline-none',
```

Comment:
I'm not sure of the specific fill value, but we just need to add `[&_svg]:fill-WHATEVER` to finish this one